Add missing Terraform install steps

jsf9k opened the pull request:
## 🗣 Description ##

This pull request modifies the Terraform installation code in `prerelease.yml` and `release.yml` to add two additional steps that were present in `build.yml` but for some reason missing in the other two files.

Resolves #55.

## 💭 Motivation and Context ##

This old code was resulting in the system Terraform (currently 0.13) being used instead of the one we manually install (currently 0.12) in the `prerelease.yml` and `release.yml` workflows.

## 🧪 Testing ##

I made the same changes in cisagov/kali-packer@953f7bc2dc5ef79723f3b909f04dd916f9764496 and verified there that they resulted in the correct `terraform` executable being used.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

jsf9k committed:
Insist that the cisagov devs are the owners of the .github directory

This additional clause must remain at the _end_ of the CODEOWNERS file
so that it cannot be overridden by a later clause.

We want to make it so that all the .github files including CODEOWNERS
are protected so only code owners (the dev team) can approve
modifications to them.

This will prevent configuration changes from breaking Actions and
other management-type functions that the files in this directory
control. By setting the .github files/folder to require code owner
approval for changes, workflow and management changes will require dev
team review and checking.

Resolves #56.

jsf9k committed:
Remove offending slash

Thanks to @dav3r and @mcdonnnj for the suggestion.

jsf9k committed:
Use the python version output by actions/setup-python

There is no need to run python code to determine the python version.
Resolves #58.

See here for details:
https://github.com/actions/setup-python/blob/main/action.yml#L14-L16

jsf9k committed:
Add missing Terraform install steps

The `build.yml` file contained these two steps, but for some reason
they were not present in the `prerelease.yml` or `release.yml` files.
This was resulting in the system Terraform (currently 0.13) being used
instead of the one we manually install (currently 0.12) in those
workflows.

Resolves #55.

mcdonnnj reviewed (approved):
🩹

dav3r reviewed (approved):
These steps are missing no more! 👣

mcdonnnj committed:
Organize pre-commit hooks

Group related pre-commit hooks together. Make sure that hooks are
alphabetically sorted within those groups.

mcdonnnj committed:
Merge pull request #60 from cisagov/organize_pre-commit_requirements

Organize and Sort pre-commit Hooks

jsf9k committed:
Merge branch 'develop' into improvement/insist-that-devs-own-gihub-dir

jsf9k committed:
Merge pull request #57 from cisagov/improvement/insist-that-devs-own-gihub-dir

Ensure that the cisagov devs are the owners of the .github directory

jsf9k committed:
Merge branch 'develop' into improvement/get-python-version-more-simply

jsf9k committed:
Merge pull request #59 from cisagov/improvement/get-python-version-more-simply

Use the python version output by actions/setup-python

jsf9k committed:
Merge remote-tracking branch 'skeleton-generic/develop' into bugfix/terraform-install-missing-steps
